content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -0,0 +1,240 @@
+    $ kubectl create namespace external
+    {{< /text >}}
+
+1.  Create a configuration file for the Squid proxy.
+
+    {{< text bash >}}
+    $ cat <<EOF > ./proxy.conf
+    http_port 3128
+
+    acl SSL_ports port 443
+    acl CONNECT method CONNECT
+
+    http_access deny CONNECT !SSL_ports
+    http_access allow localhost manager
+    http_access deny manager
+    http_access allow all
+
+    coredump_dir /var/spool/squid
+    EOF
+    {{< /text >}}
+
+1.  Create a Kubernetes [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
+    to hold the configuration of the proxy:
+
+    {{< text bash >}}
+    $ kubectl create configmap proxy-configmap -n external --from-file=squid.conf=./proxy.conf
+    {{< /text >}}
+
+1.  Deploy a container with Squid:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: squid
+      namespace: external
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: squid
+        spec:
+          volumes:
+          - name: proxy-config
+            configMap:
+              name: proxy-configmap
+          containers:
+          - name: squid
+            image: sameersbn/squid:3.5.27
+            imagePullPolicy: IfNotPresent
+            volumeMounts:
+            - name: proxy-config
+              mountPath: /etc/squid
+              readOnly: true
+    EOF
+    {{< /text >}}
+
+1.  Deploy the [sleep]({{< github_tree >}}/samples/sleep) sample in the `external` namespace to test traffic to the
+    proxy without Istio traffic control.
+
+    {{< text bash >}}
+    $ kubectl apply -n external -f @samples/sleep/sleep.yaml@
+    {{< /text >}}
+
+1.  Define an environment variable to hold the IP address of the proxy pod:
+
+    {{< text bash >}}
+    $ export PROXY_IP=$(kubectl get pod -n external -l app=squid -o jsonpath={.items..podIP})
+    {{< /text >}}
+
+1.  Define an environment variable to hold the port of your proxy. The deployment of Squid in this example uses port
+    3128.
+
+    {{< text bash >}}
+    $ export PROXY_PORT=3128
+    {{< /text >}}
+
+1.  Send a request from the `sleep` pod in the `external` namespace to an external service via the proxy:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -n external -l app=sleep -o jsonpath={.items..metadata.name}) -n external -- sh -c "HTTPS_PROXY=$PROXY_IP:$PROXY_PORT curl https://en.wikipedia.org/wiki/Main_Page" | grep -o "<title>.*</title>"
+    <title>Wikipedia, the free encyclopedia</title>
+    {{< /text >}}
+
+1.  Check the access log of the proxy for your request:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -n external -l app=squid -o jsonpath={.items..metadata.name}) -n external -- tail -f /var/log/squid/access.log
+    1544160065.248    228 172.30.109.89 TCP_TUNNEL/200 87633 CONNECT en.wikipedia.org:443 - HIER_DIRECT/91.198.174.192 -
```

Comment:
Please do not include the output in the same code block as the command and without any explanation or context for the reader. In this specific instance, I don't think you need to include the output but, if you want to explain the output, that would be welcome. Just make sure you provide an explanation to the readers.